vms/avm/genesis_builder.go

```diff
@@ -88,7 +80,7 @@ type BuildGenesisReply struct {
 
 // BuildGenesis returns the UTXOs such that at least one address in [args.Addresses] is
 // referenced in the UTXO.
-func (*StaticService) BuildGenesis(_ *http.Request, args *BuildGenesisArgs, reply *BuildGenesisReply) error {
+func BuildGenesis(args *BuildGenesisArgs, reply *BuildGenesisReply) error {
```

Comment:
Can we also get rid of all these args/reply structs and just make them parameters/return values/structs?

Reply:
yep good point, i'll update this